api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/CommonCohortQueries.java

```diff
@@ -504,7 +504,7 @@ public CohortDefinition getMOHPatientsOnTreatmentFor6Months(
       query +=
           "             AND o.value_coded IS NOT NULL  "
               + "             AND DATE(o.obs_datetime) <= DATE(clinical.last_visit)  "
-              + "         AND DATEDIFF(clinical.last_visit, o.obs_datetime) >= 180  "; // check
+              + "         AND DATE(o.obs_datetime) <= DATE_SUB(clinical.last_visit,INTERVAL 6 MONTH) "; // check
```

Comment:
Not sure if 6 months is equivalent to 180 days, 6 calendar months are different from the days, please confirm

Reply:
Yes there is a slight difference for SQL, hence the result using days to calculate was not matching FGH's results in this particular indicator, using SQL MONTH instead, matches the results.

Reply:
Mhhh, that is fine then, please ressolve the confilcts, then I will merge this